business_requirement_deliverable/models/business.py

```diff
@@ -0,0 +1,272 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+from openerp.exceptions import ValidationError, Warning
+from openerp.tools.translate import _
+
+
+class BusinessRequirementResource(models.Model):
+    _name = "business.requirement.resource"
+    _description = "Business Requirement Resource"
+
+    sequence = fields.Integer('Sequence')
+    description = fields.Char('Description', required=True)
+    product_id = fields.Many2one(
+        comodel_name='product.product',
+        string='Product',
+        required=False
+    )
+    uom_id = fields.Many2one(
+        comodel_name='product.uom',
+        string='UoM',
+        required=True
+    )
+    qty = fields.Float(
+        string='Quantity',
+        default=1,
+    )
+    resource_type = fields.Selection(
+        selection=[('task', 'Task'), ('procurement', 'Procurement')],
+        string='Type',
+        required=True,
+        default='task'
+    )
+    user_id = fields.Many2one(
+        comodel_name='res.users',
+        string='Assign To',
+        ondelete='set null'
+    )
+    business_requirement_deliverable_id = fields.Many2one(
+        comodel_name='business.requirement.deliverable',
+        string='Business Requirement Deliverable',
+        ondelete='cascade'
```

Comment:
@dreispt IMO we should maintain as `cascade`.
 
Because the `business.requirement.deliverable` records only have meaning when belongs to an existing `business.requirement` record.

If `set null` and later delete a "business requirement" (which has at least on deliverable) we will have an increasing number of "deliverable" records inaccessible.